src/main/java/net/snowflake/client/core/SessionUtilExternalBrowser.java

```diff
@@ -256,6 +256,10 @@ private String generateProofKey() {
     return Base64.getEncoder().encodeToString(randomness);
   }
 
+  private int getBrowserResponseTimeout() {
+    return loginInput.getBrowserResponseTimeout() * 1000;
```

Comment:
we are converting here from seconds to millis - please use Duration class where possible